.travis.yml

```diff
@@ -7,12 +7,25 @@ node_js:
 # dist: xenial
 # os: linux
 
+before_install:
+  - sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/11/main/postgresql.conf
```

Comment:
this seems to all be needed to make sure auth stuff is working becuase we're installing a postgres version that travis dosen't have installed